src/controllers/album.js

```diff
@@ -0,0 +1,92 @@
+const db = require("../db/index");
+
+const createAlbum = async (req, res) => {
+  const { id: artistId } = req.params;
+  const { name, year } = req.body;
+
+  try {
+    const {
+      rows: [album],
+    } = await db.query(
+      `INSERT INTO Albums (name, year, artistId) VALUES ($1, $2, $3) RETURNING *`,
+      [name, year, artistId]
+    );
+    res.status(201).json(album);
+  } catch (err) {
+    res.status(500).json(err.message);
+  }
+};
+
+const readAlbums = async (req, res) => {
```

Comment:
if you aren't using a parameter you can replace it with an underscore `_` to show it is unused. If you aren't using multiple parameters you can start each unused one with an underscore to show they're unused, e.g. `_req`